unyt/array.py

```diff
@@ -2125,6 +2125,11 @@ def dot(self, b, out=None):
             out.units = res_units
         return ret
 
+    def take(self, *args, **kwargs):
+        from ._array_functions import take as unyt_take
+
+        return unyt_take(self, *args, **kwargs)
```

Comment:
aliasing isn't necessary
```suggestion
        from ._array_functions import take

        return take(self, *args, **kwargs)
```